lib/webinspector/index.js

```diff
@@ -6,24 +6,41 @@ import PlistServiceEncoder from '../plist-service/transformer/plist-service-enco
 import LengthBasedSplitter from '../util/transformer/length-based-splitter';
 import _ from 'lodash';
 import { MB } from '../constants';
+import log from '../logger';
+
 
 const WEB_INSPECTOR_SERVICE_NAME = 'com.apple.webinspector';
 
 const PARTIAL_MESSAGE_SUPPORT_DEPRECATION_VERSION = 11;
 
 const MAX_FRAME_SIZE = 20000000;
 
+function getPlistString (plist) {
+  // Buffers cannot be serialized in a readable way
+  const bufferToJSON = Buffer.prototype.toJSON;
+  delete Buffer.prototype.toJSON;
```

Comment:
why this is necessary if we anyway transform the buffer to a string later?

Reply:
Because the transformation happens elsewhere, after the calls in this function. The alternative to this would be to pass the `verbose` parameter through all the transforms to the place where it is all serializable. 

I'm moving this function, though, to `appium-support` for reusability.